### DISCUSSION TITLE
[Script request] rclone + GUI + shares

### DISCUSSION THREAD

**Discussion by jsapede** [community-scripts/ProxmoxVE#673] [Request script] [answered]
### Application Name

rclone

### Website

https://rclone.org/

### Description

Rclone is a command-line program to manage files on cloud storage. It is a feature-rich alternative to cloud vendors' web storage interfaces. [Over 70 cloud storage products](https://rclone.org/#providers) support rclone including S3 object stores, business & consumer file storage services, as well as standard transfer protocols.

Rclone has powerful cloud equivalents to the unix commands rsync, cp, mv, mount, ls, ncdu, tree, rm, and cat. Rclone's familiar syntax includes shell pipeline support, and --dry-run protection. It is used at the command line, in scripts or via its [API](https://rclone.org/rc).

Users call rclone "The Swiss army knife of cloud storage", and "Technology indistinguishable from magic".

would be useful to have installation inside existing lxc as well as standalone with capabilites to add shared folders with other lxc 
GUI as extra !

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by tremor021** [comment]
```bash
wget https://github.com/rclone/rclone/releases/download/v1.69.1/rclone-v1.69.1-linux-amd64.de
dpkg -i rclone-v1.69.1-linux-amd64.deb
rclone rcd --rc-web-gui --rc-web-gui-no-open-browser --rc-addr :3000 --rc-user admin --rc-pass 12345
```
Login at IP:3000

Do we really need a script for 3 bash commands?

**Reply by ilkka** [reply]
I don't think it's the complexity, it's the discoverability and ~~the fact~~ that it's nice that all things work the same way. I'm surely not alone in treating the script repo like "the pve app store" and mostly not bothering with stuff that's not on there.

**Comment by tremor021** [comment, marked as answer]
https://community-scripts.github.io/ProxmoxVE/scripts?id=rclone
